src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
Consider adding the full styles from the previous task here to keep modifiers working